### PR TITLE
Adds more polychromic clothes to clothesmate

### PR DESCRIPTION
**Pull request by QuacksQ**
## About The Pull Request

Adds some more polychormic clothes to the clothesmate that were previously not there.

## How This Contributes To The Skyrat Roleplay Experience

More clothes is always good and especially these since its alot more freedom for players to customize their characters with the clothes and colors that are to their liking.

## Changelog

:cl:
add:Added more polychromic clothes to the Clothesmate.
/:cl:

### CODE REVIEW

**Comment by YakumoChen** [comment]
Some of these are/were on the lewd vendor, such as bottomless clothes, panties, etc. If they aren't they should probably be returned to it.

Also you should sort these a bit to be in the same place on the list as the rest of the polyclothes.

File: modular_skyrat/modules/modular_vending/code/clothesmate.dm [hunk 1]
```diff
@@ -84,6 +84,13 @@
 		/obj/item/clothing/head/fedora/white = 5,
 		/obj/item/clothing/head/fedora/fedblack = 5,
 		/obj/item/clothing/head/fedora/fedbrown = 5,
+		/obj/item/clothing/suit/jacket/leather/polychromic = 5,
+		/obj/item/clothing/under/misc/polyshorts = 5,
+		/obj/item/clothing/under/misc/polyjumpsuit = 5,
```
Comment (Zonespace27) [inline modular_skyrat/modules/modular_vending/code/clothesmate.dm]: already in the clothesmate

File: modular_skyrat/modules/modular_vending/code/clothesmate.dm [hunk 2]
```diff
@@ -84,6 +84,13 @@
...
+		/obj/item/clothing/under/misc/poly_bottomless = 5,
```
Comment (Zonespace27) [inline modular_skyrat/modules/modular_vending/code/clothesmate.dm]: Mixed feelings on this being in the clothesmate at all

File: modular_skyrat/modules/modular_vending/code/clothesmate.dm [hunk 3]
```diff
@@ -84,6 +84,13 @@
...
+		/obj/item/clothing/under/misc/polysweater = 5,
+		/obj/item/clothing/under/shorts/polychromic = 5,
+		/obj/item/clothing/under/shorts/polychromic/pantsu = 5,
```
Comment (Zonespace27) [inline modular_skyrat/modules/modular_vending/code/clothesmate.dm]: ditto

**Review by Zonespace27 — CHANGES_REQUESTED** [review]

File: modular_skyrat/modules/modular_vending/code/clothesmate.dm (hunk 3 above)
Comment (QuacksQ) [reply modular_skyrat/modules/modular_vending/code/clothesmate.dm:93]: Ditto?

File: modular_skyrat/modules/modular_vending/code/clothesmate.dm (hunk 1 above)
Comment (QuacksQ) [reply modular_skyrat/modules/modular_vending/code/clothesmate.dm:89]: I dont know where you're seeing them in the clothesmate but I cant find them being in there at all. I might be blind though *shrug

**Comment by QuacksQ** [comment]
Removed the duplicates and the stuff that's not supposed to be there and placed it with the rest of the poly clothes now
Should I add the lewd stuff to the vendor for that or just leave it be?

**Comment by YakumoChen** [comment]
add the lewd stuff to the lewd vendor. I'm very sure it was added at some point but someone else removed them for no reason.

**Comment by YakumoChen** [comment]
found it. #8457 removed the clothing additions, probably due to a merge conflict.

**Comment by YakumoChen** [comment]
`					/obj/item/clothing/under/shorts/polychromic/pantsu = 4,
					/obj/item/clothing/under/misc/poly_bottomless = 4,
					/obj/item/clothing/under/misc/poly_tanktop = 4,
					/obj/item/clothing/under/misc/poly_tanktop/female = 4,`

all should be readded to the torso section of lustwish

**Review by Zonespace27 — APPROVED** [review]

**Comment by QuacksQ** [comment]
🔨 Dont close my pr, Github please.

**Pull request closed without merging**